commands/build.go

```diff
@@ -43,6 +44,10 @@ import (
 	"google.golang.org/grpc/codes"
 )
 
+const ImageSourceLabel = "org.opencontainers.image.source"
+const ImageRevisionLabel = "org.opencontainers.image.revision"
+const DockerfileLabel = "com.docker.image.dockerfile.path"
```

Comment:
Looking at the slsa definition maybe this should be `com.docker.image.source.entrypoint` . Might be better to avoid Dockerfile just because it doesn't actually need to be a Dockerfile as Buildkit supports many formats.

Reply:
Sounds good to me.